README.md

```diff
@@ -38,6 +38,12 @@ git submodule update --init --recursive
 
 Then set up [git secrets](#git-secrets) and fire up the [development servers](#running-the-dev-servers). Optionally, you can [set up your Intellij](https://docs.google.com/document/d/1DtESBapEzvuti7xODTFPHorwmLM7LybF-6D5lhbIkLU/edit) for UI or API work.
 
+Before doing any development, you must run:
+```Shell
+./gradlew compileGeneratedJava appengineRun
```

Comment:
is `compileGeneratedJava` necessary? I believe its a task that is dependent on `appengineRun` so it should be run regardless.

Also, we should indicate that this command runs form the `api`.